tsconfig.json

```diff
@@ -21,6 +21,12 @@
       "@/*": ["./app/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    ".next/types/**/*.ts",
+    "./postcss.config.js"
```

Comment:
When this was not written here, it casues an error about way to use module in postcss.config.js